src/backend/ProcessLauncher.cpp

```diff
@@ -152,7 +152,7 @@ void ProcessLauncher::onLaunchRequested(const model::GameFile* q_gamefile)
     workdir = helpers::abs_workdir(workdir, game.relative_basedir, default_workdir);
 
 
-    beforeRun();
+    beforeRun({gamefile.fileinfo.absolutePath()});
```

Comment:
`beforeRun` should take a single `QString` parameter, which then gets turned into...